packages/portfolio-contract/DESIGN-BETA.md

```diff
@@ -162,6 +162,55 @@ sequenceDiagram
     participant Res as Resolver
   end
 
+  box rgb(163,180,243) Noble
+      participant icaN as icaNoble
+  end
+  box rgb(163,180,243) Axelar
+      participant AX as GMP
+  end
+  box rgb(163,180,243) Arbitrum
+      participant pc as CCTPProxyContract
+      participant acctArb as acctArb*
+      participant aavePos as aavePos*
+  end
+  box rgb(163,180,243) Base
+      participant acctBase as acctBase*
+      participant compPos as compPos*
+  end
+
+  %% get plan
+  UI ->> portfolio: withdraw with steps, <br>steps also include CCTP fee and proxy contract address
```

Comment:
proxy contract address? That's news to me. why?

Reply:
Yes, it should be in the contract configuration